services/searchengine/bleveengine/search.go

```diff
@@ -209,6 +212,72 @@ func (b *BleveEngine) SearchPosts(channels *model.ChannelList, searchParams []*m
 	return postIds, matches, nil
 }
 
+func (b *BleveEngine) DeletePosts(searchRequest *bleve.SearchRequest, batchSize int) (int64, error) {
```

Comment:
nit: does this method need to be public?

Reply:
Not necessary but in case we want to start using it instead of creating specialized methods could be useful

Reply:
As the method receives a `bleve.SearchRequest` object, I don't think is very likely that we use it  from outisde of the engine, but I'm really 1/5 on this, so if you feel strongly about it, I'm happy with the method being public :)

Reply:
Hum, I see your point. Ok, I'll make it private :)